code/modules/projectiles/projectile_parent.dm

```diff
@@ -50,7 +50,7 @@
 	var/goes_through_walls = 0
 	var/goes_through_mobs = 0
 	var/collide_with_other_projectiles = 0 //allow us to pass canpass() function to proj_data as well as receive bullet_act events
-	var/list/hitlist //list of atoms collided with this tick
+	var/list/hitlist = list() //list of atoms collided with this tick
```

Comment:
Not that relevant now since a bunch of other lists are initialized too but it might be worth it to keep it at `null` and initialize the list lazily because setting it to `list()` in the definition adds a hidden `(init)` proc to the object that's ran before `New`. (Imo we should just straight up make macros for working with such lazy lists.)